ash/src/vk.rs

```diff
@@ -46,21 +72,153 @@ pub(crate) unsafe fn ptr_chain_iter<T: ?Sized>(
         Some(old)
     })
 }
-pub trait Handle: Sized {
-    const TYPE: ObjectType;
-    fn as_raw(self) -> u64;
-    fn from_raw(_: u64) -> Self;
 
-    /// Returns whether the handle is a `NULL` value.
-    ///
-    /// # Example
-    ///
-    /// ```
-    /// # use ash::vk::{Handle, Instance};
-    /// let instance = Instance::null();
-    /// assert!(instance.is_null());
-    /// ```
-    fn is_null(self) -> bool {
-        self.as_raw() == 0
+#[cfg(test)]
+mod tests {
```

Comment:
nit: place at the bottom of the module?

Reply:
Done. Wanted to keep it close to `fn ptr_chain_iter()` which is the only thing that is being tested; for that we could've dropped the module entirely though because there's nothing else to test.

Reply:
Actually we also have a very minimal `ash/tests/display.rs` that we could inline into `vk.rs` where the `fn debug_flags()` at test is defined?

Reply:
Sure, if you like.

Reply:
Decided to keep the `mod` as it seems more fitting with multiple `#[test]` functions (unless you think those individual test functions should move below the actual implementations), and because I was extending the `push()` tests with more missed validations at #971.